wrapping/itkTileMergeImageFilter.wrap

```diff
@@ -1,10 +1,13 @@
 itk_wrap_class("itk::TileMergeImageFilter" POINTER)
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
     foreach(t ${WRAP_ITK_SCALAR})
-      itk_wrap_template("${ITKM_I${t}${d}}" "${ITKT_I${t}${d}}")
+      itk_wrap_template("${ITKM_I${t}${d}}" "${ITKT_I${t}${d}}, double")
```

Comment:
Just double checking, is it intended that the mangling does not change so that the class interface is preserved? Or should "${ITKM_I${t}${d}}" become "${ITKM_I${t}${d}}D" ?

Reply:
The class will indeed not be the same. But does that matter? It is not expected that versions of montage will be mixed somehow. This change should not be externally visible, as we are only changing internal accumulation precision during processing.

Reply:
I agree, not really a big deal for this change. In general it seems to be a good practice to align Python manglings with C++ template parameters, especially if there were ever changes down the line to expose other template parameters such as the interpolator in the Python interface. But it sounds like lower-precision accumulator types shouldn't really be used anyway, so I'm fine with not worrying about it here.